lib/greet.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+module.exports = exports = {};
+var name = process.argv[2];
+
+exports.sayHi = function(name) {
+  if(arguments.length === 0) throw new Error('name not provided');
+  console.log(`Hi ${name}!`);
+  return `Hi ${name}!`;
+};
+
+exports.saySeeYaLater = function(petName) {
+  if(arguments.length === 0) throw new Error('pet name not provided');
+  console.log(`See ya later, ${petName}!`);
+  return `See ya later, ${petName}!`;
+};
+
+exports.sayHi(name);
+exports.saySeeYaLater('Gaara');
```

Comment:
Also, instead of if(arguments.length === 0) try using if(!name) instead. This is a much more common practice and something we will be using a lot in class.